pkg/webui/console/views/gateway-collaborator-edit/gateway-collaborator-edit.styl

```diff
@@ -0,0 +1,17 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.right-label
```

Comment:
just a remainder
after https://github.com/TheThingsNetwork/lorawan-stack/pull/817 it wont be needed